src/app/app.scss

```diff
@@ -0,0 +1,200 @@
+@import 'src/sass/mixins/index';
+@import 'src/sass/variables/index';
+
+.App {
+  display: block;
+  &-main {
+    width: 100%;
+    background-color: $primary-color;
+    display: flex;
+    flex-wrap: nowrap;
+    @media screen and (max-width: 1216px) {
+      flex-wrap: wrap;
+    }
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    &-left,
+    &-right {
+      margin: 2rem;
+    }
+  }
+  &-title {
+    font-size: 3rem;
+    font-weight: 400;
+    letter-spacing: 1px;
+    color: white;
+    margin-bottom: 1rem;
+    text-align: left;
+    @media screen and (max-width: 1216px) {
+      text-align: center;
+    }
+  }
+  &-description {
+    color: white;
+    font-weight: 400;
+    font-size: 1.15rem;
+    max-width: 50ch;
+    text-align: left;
+    @media screen and (max-width: 1216px) {
+      text-align: center;
+    }
+  }
+  &-screenshot {
+    max-width: 832px;
+    width: 100%;
+    box-shadow: 0 8px 38px 3px rgba(0, 0, 0, 0.2);
+  }
+  &-download {
+    @include button(white);
+    color: $primary-color;
+    transition: box-shadow $transition;
+    clear: both;
+    float: left;
+    margin-bottom: 1rem;
+    @media screen and (max-width: 1216px) {
+      transform: translateX(-50%);
+      position: relative;
+      left: 50%;
+    }
+    > .nc-icon {
+      margin-left: 0;
+      margin-right: 1rem;
+    }
+    &:hover {
+      box-shadow: 0 0 10px 3px rgba(255, 255, 255, 0.2);
+    }
+  }
+  &-other-links {
```

Comment:
Bleeds off screen on really thin phones (e.g. iPhone 5S) I'd either kill the whole element, or make the links stack vertically.